packages/platform-ios/src/commands/logIOS/index.ts

```diff
@@ -15,7 +15,7 @@ import {Device} from '../../types';
 function findAvailableDevice(devices: {[index: string]: Array<Device>}) {
   for (const key of Object.keys(devices)) {
     for (const device of devices[key]) {
-      if (device.availability === '(available)' && device.state === 'Booted') {
+      if ((device.availability === '(available)' || device.isAvailable) && device.state === 'Booted') {
```

Comment:
We have similar logic in the `findMatchingSimulator.ts` file. I just noticed that we even call  the same command (`simctl list devices --json`). Can you think of a way to reuse availability check? Can be done in a followup, no worries :)